purchase_custom_search_product/__init__.py

```diff
@@ -0,0 +1,2 @@
+# -*- coding: utf-8 -*-
+from . import models
```

Comment:
missing eol and unnecessary encoding line.

Reply:
ok